packages/rax/src/server/serializer.js

```diff
@@ -116,6 +118,13 @@ class Serializer {
   }
 
   serialize() {
+    if (this.needAddCheckSum) {
+      // add checksum to the rootChilds
+      Array.prototype.forEach.call(this.startNode.childNodes, (rootChildNode, index) => {
+        this.startNode.childNodes[index] = addChecksumToElement(rootChildNode);
+      });
+    }
```

Comment:
Add checksum in serialize has a little ugly. What about migrate to `renderToString.js`, between `render` and `Serializer`?